### PR TITLE
[REF] supported matic tokens updates

### PR DESCRIPTION
**Pull request by rastajpa**
updates are based on this list:
Captura de Pantalla 2022-10-04 a la(s) 15 11 05

### CODE REVIEW

File: packages/bitcore-wallet-client/src/lib/common/constants.ts [hunk 1]
```diff
@@ -55,16 +55,13 @@ export const Constants = {
...
     'usdc_m',
-    'pax_m',
-    'gusd_m',
     'busd_m',
     'dai_m',
     'wbtc_m',
     'shib_m',
     'ape_m',
-    'euroc_m'
+    'matic_m'
```
Comment (escottalexander) [inline packages/bitcore-wallet-client/src/lib/common/constants.ts:64]: I guess this is wrapped MATIC?

File: packages/crypto-wallet-core/src/constants/tokens.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ export let ETH_TOKEN_OPTS = {
   '0x2260fac5e5542a773aa44fbcfedf7c193bc2c599': {
     name: 'Wrapped Bitcoin',
     symbol: 'WBTC',
-    decimal: 9,
+    decimal: 8,
```
Comment (escottalexander) [inline packages/crypto-wallet-core/src/constants/tokens.ts:35]: Good catch on the decimal number. Shouldn't this be `decimals` per your other changes?

**Review by escottalexander — CHANGES_REQUESTED** [review]
These changes look good to me except for the one thing I mentioned.

File: packages/bitcore-wallet-client/src/lib/common/constants.ts (hunk 1 above)
Comment (rastajpa) [reply packages/bitcore-wallet-client/src/lib/common/constants.ts:64]: MATIC Token is its name

File: packages/crypto-wallet-core/src/constants/tokens.ts (hunk 2 above)
Comment (rastajpa) [reply packages/crypto-wallet-core/src/constants/tokens.ts:35]: yes, you're right! ty!

File: packages/bitcore-wallet-client/src/lib/common/constants.ts (hunk 1 above)
Comment (escottalexander) [reply packages/bitcore-wallet-client/src/lib/common/constants.ts:64]: In that case I don't think this is needed. MATIC is the base asset on the Polygon chain similar to how ETH is the base asset on the Ethereum chain. We don't need to add it as a token here.

File: packages/crypto-wallet-core/src/constants/tokens.ts [hunk 3]
```diff
@@ -2,124 +2,106 @@ export let ETH_TOKEN_OPTS = {
...
-  '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619': {
-    name: 'Wrapped Ethereum',
-    symbol: 'ETH',
-    decimal: 18,
-    address: '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619'
+  '0x0000000000000000000000000000000000001010': {
+    name: 'Matic Token',
+    symbol: 'MATIC',
+    decimals: 18,
+    address: '0x0000000000000000000000000000000000001010'
```
Comment (escottalexander) [inline packages/crypto-wallet-core/src/constants/tokens.ts:105]: Per my above comment we don't need this either.

File: packages/bitcore-wallet-client/src/lib/common/constants.ts (hunk 1 above)
Comment (rastajpa) [reply packages/bitcore-wallet-client/src/lib/common/constants.ts:64]: in the pr description image is there a list of all the tokens that we support, `matic_m` is simply a token in the polygon chain called "MATIC Token" different from the MATIC coin that would be the equivalent of ETH in its own chain

**Review by escottalexander — APPROVED** [review]
These look good to me.

**Comment by rastajpa** [comment]
> These look good to me.

I apreciate it. I did a master rebase to see if the checks pass

**Comment by rastajpa** [comment]
@escottalexander I wanted to let you know that I temporarily disabled the Matic token as you suggested earlier because this was only causing errors. For example, in the app when adding that token we obtained the same balance as in the Matic coin, all this speaking only of the Polygon network... a bit confusing, it is also very difficult to differentiate the token from the coin because the coin and chain parameters they are exactly the same, the only difference is the tokenaddress in the token.
if in the future it is necessary to add this confusing token we will see how to solve it from the bitcore side and from the app side

**Review by kajoseph — APPROVED** [review]

**Review by kajoseph — DISMISSED** [review]

**Pull request merged**